src/main/java/xyz/nucleoid/plasmid/storage/ServerStorage.java

```diff
@@ -36,11 +36,27 @@
      * Called when saving this {@link ServerStorage} to NBT.
      * @return - The serialized NBT data of this {@link ServerStorage}.
      */
-    NbtCompound toTag();
+    NbtCompound writeNbt();
```

Comment:
This breaking might be unavoidable, given classes previously implementing this type would now no longer compile due to missing implementations for the new methods. We might only be able to get around it by making all the serialization functions default, but that's not ideal.